analytic_base_department/tests/test_line_department.py

```diff
@@ -0,0 +1,25 @@
+# Copyright 2016 Yannick Vaucher (Camptocamp)
+# License AGPL-3.0 or later (http://www.gnu.org/licenses/agpl).
+from odoo.tests.common import TransactionCase
+
+
+class LineDepartmentCase(TransactionCase):
+    def setUp(self):
+        """Assign user and department."""
+        super(LineDepartmentCase, self).setUp()
+        # base.user_demo --> hr.employee_qdp --> hr.dep_rd
+        self.user = self.env.ref("base.user_demo")
+        self.dep = self.env.ref("hr.dep_rd")
```

Comment:
This should be now [`setUpClass`](https://github.com/odoo/odoo/blob/d4e7eb99818cd6780c3e0a94a9045951ac3708a7/odoo/tests/common.py#L698).

```suggestion
    @classmethod
    def setUpClass(cls):
        """Assign user and department."""
        super(LineDepartmentCase, cls).setUpClass()
        # base.user_demo --> hr.employee_qdp --> hr.dep_rd
        cls.user = cls.env.ref("base.user_demo")
        cls.dep = cls.env.ref("hr.dep_rd")
```